server/routes/creatingLicences/handlers/checkAnswers.test.ts

```diff
@@ -113,6 +116,59 @@ describe('Route Handlers - Create Licence - Check Answers', () => {
       expect(res.render).toHaveBeenCalledWith('pages/create/checkAnswers', {
         additionalConditions: [],
         conditionsWithUploads: [],
+        isInPssPeriod: false,
+        backLink: req.session.returnToCase,
+      })
+      expect(licenceService.recordAuditEvent).toHaveBeenCalled()
+    })
+
+    it('should render view, record audit event (not owner) and PSS period should be true', async () => {
```

Comment:
not sure you need this test anymore? 

pssPeriod presumably will be read from the licence directly rather than calculated so don't need a specific test for checking its value?